builders/toolkit/integrations/wallets/rainbowkit.md

```diff
@@ -0,0 +1,316 @@
+
+
+RainbowKit bundles together mulitple tools to simplify adding wallet connection to your dApp: 
+
+- [Wagmi](https://wagmi.sh/){target=\_blank} - a React Hooks library for interacting with Ethereum accounts, wallets, contracts, transactions, signing, ENS, and more
+- [viem](https://viem.sh/){target=\_blank} - TypeScript interface which provides low-level stateless primitives for interacting with Ethereum
+- [WalletConnect](https://walletconnect.com/){target=\_blank} - adds encrypted connections and enhanced UX experiences like connecting a mobile wallet by scanning a QR code
+- [TanStack Query](https://tanstack.com/query/latest/docs/framework/react/overview){target=\_blank} - helps manage and update server state within the application
+
+This guide takes you through the process of adding RainbowKit to a dApp using the CLI, supporting Tanssi appchains,and some options for further customizing your integration. 
+
+## Quick Start {: #quick-start }
+
+If you are starting a new project, RainbowKit can scaffold a project from the CLI, combining RainbowKit and Wagmi in a [Next.js](https://nextjs.org/docs){target=\_blank} application. Use your package manager of choice to run the CLI command and start your project:
+
+=== "npm"
+
+    ```bash
+    npm init @rainbow-me/rainbowkit@latest 
+    ```
+
+=== "pnpm"
+
+    ```bash
+    pnpm create @rainbow-me/rainbowkit@latest
+    ```
+=== "yarn"
+
+    ```bash
+    yarn create @rainbow-me/rainbowkit
+    ```
+
+The script will prompt you for a project name, generate a new directory with the boilerplate starter code, and install all required dependencies. 
+
+--8<-- 'code/builders/toolkit/integrations/wallets/rainbowkit/cli-quickstart.md'
+
+You can now navigate to the project directory, start the development server, and navigate to [http://localhost:3000](http://localhost:3000){target=\_blank} to view your project locally:
+
+=== "npm"
+
+    ```bash
+    cd INSERT_PROJECT_NAME
+    npm run dev 
+    ```
+
+=== "pnpm"
+
+    ```bash
+    cd INSERT_PROJECT_NAME
+    pnpm run dev
+    ```
+
+=== "yarn"
+
+    ```bash
+    cd INSERT_PROJECT_NAME
+    yarn dev
+    ```
+
+Your starting screen should look like this:
+
+![Scaffolded RainbowKit project landing page](/images/builders/toolkit/integrations/wallets/rainbowkit/rainbowkit-1.webp)
+
+Open the project in your code editor and take a look at the directory and file structure, paying particular attention to the `wagmi.ts` file. This file is where you can configure which chains are included in the list of networks that users can connect to through your dApp.
+
+Since Dancebox is a custom appchain on Tanssi, it cannot be imported directly from `wagmi/chains`. Instead, you need to manually define the chain in the `wagmi.ts` file. For example, if your appchain uses the Tanssi demo appchain (Dancebox) or another EVM-compatible chain, you can add the necessary configurations manually.
```

Comment:
```suggestion
Since the Dancebox EVM demo appchain is a custom appchain on Tanssi, it cannot be imported directly from `wagmi/chains`. Instead, you must manually define the chain in the `wagmi.ts` file. For example, if your appchain uses the Tanssi demo appchain (Dancebox) or another EVM-compatible chain, you can add the necessary configurations manually.
```